designsystem/src/commonMain/kotlin/org/hisp/dhis/mobileui/designsystem/component/InputField.kt

```diff
@@ -16,36 +16,74 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.geometry.CornerRadius
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.PathEffect
 import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.text.input.VisualTransformation
 import org.hisp.dhis.mobileui.designsystem.theme.Outline
 import org.hisp.dhis.mobileui.designsystem.theme.Radius
 import org.hisp.dhis.mobileui.designsystem.theme.Spacing
 import org.hisp.dhis.mobileui.designsystem.theme.SurfaceColor
+import org.hisp.dhis.mobileui.designsystem.theme.TextColor
 
 /**
- * DHIS2 Text Input Field. Wraps Material· [BasicTextField].
+ * DHIS2 EmptyInput
+ * empty input container with dashed border
+ * used for internal test purposes
+ */
+@Composable
+fun EmptyInput(
+    modifier: Modifier = Modifier
+) {
+    Row(modifier = modifier.fillMaxWidth().height(Spacing.Spacing24).background(SurfaceColor.SurfaceBright)) {
+        val stroke = Stroke(
+            width = 2f,
+            pathEffect = PathEffect.dashPathEffect(floatArrayOf(10f, 10f), 0f)
+        )
+        Canvas(Modifier.fillMaxWidth().height(Spacing.Spacing24)) {
+            drawRoundRect(color = Outline.Light, style = stroke, cornerRadius = CornerRadius(x = Radius.XS.toPx(), y = Radius.XS.toPx()))
+        }
+    }
+}
+
+/**
+ * DHIS2 Basic Input. Wraps Material· [BasicTextField].
  * @param enabled Controls the enabled state of the component. When `false`, this component will not be
  * clickable and will appear disabled to accessibility services.
+ * @param helper Manages the helper text to be shown
+ * @param helperStyle manages the helper text style, NONE by default
  */
 @Composable
-fun SimpleTextInputField(
-    enabled: Boolean = true
+fun BasicInput(
+    helper: String? = null,
+    enabled: Boolean = true,
+    helperStyle: InputStyle = InputStyle.NONE
 ) {
-    var text by remember { mutableStateOf("") }
+    var text by remember { mutableStateOf("Input") }
```

Comment:
We should about hard coded text

Reply:
yes you are right I set it up like that for the example and it passe me by